### ISSUE TITLE
history can't be used for a second action

### ISSUE THREAD

**Issue by Diginix** [nostalgy/nostalgy#91]
If I move an email to a folder and after that I want to go there or move a second email to the same folder, I always have to type again the folders name to be able to select it. Even it is listed in the hostory above the input, I can't select any entry of the history. Neither with mouse nor with cursor keys.
TB 68.9.0
Nostalgy 1.0.21 (xpi from 2020-01-03)

**Comment by Olifair** [comment]
I think this is related to issue #54 or #55

**Comment by Diginix** [comment]
@Olifair Maybe, but this is not 100% clear. And the issue with me is only with the Nostalgy fork for TB68.
With Nostalgy 0.3 in TB 52.9 the history navigation and entry selection is working fine. So, it could be a bug or a missing implementation in version 1.x.

**Comment by opto** [comment]
I think the history works if you do left arrow, then you can scroll in the history.

**Comment by rmmbr** [comment]
> I think the history works if you do left arrow, then you can scroll in the history.

Left is no workaround on my System ... I tried a lot. Every "s" requieres a complete new search ...

For the same folder I use SHIFT+[s, g].

Windows 10 1909 x64 (aktuell), Thunderbird 68.9.0, Nostalgy 0.3.

**Comment by opto** [comment]
yes, TB has changed the way how S is used (S sets star on emails, and pressing S is handled differently by TB now as compared to pre TB 59.). Also, TB autocomplete has changed as how to create/fill the initial box with the history. 

It might help to set another character, one which is not in use by TB (just press a few letters by trial and error and choose one that does nothing). I am updating for 78, and history will/will not work there, depending how the TB core is changed.

Klaus

**Comment by eliotmoss** [comment]
On 6/30/2020 1:41 AM, rmmbr wrote:
>     I think the history works if you do left arrow, then you can scroll in the history.
> 
> Left is no workaround on my System ... I tried a lot. Every "s" requieres a complete new search ...
> 
> For the same folder I use SHIFT+[s, g].
> 
> Windows 10 1909 x64 (aktuell), Thunderbird 68.9.0, Nostalgy 0.3.

Interesting.  For me, left, then up, pops up the menu within which one can scroll ...

EM

**Comment by Olifair** [comment]
Ah great, don't know why I hadn't tried this, probably because when hitting "s", the menu pops up and when hitting "<-" it disappears again, but when then hitting "up" it comes back !!! Great !!!! This will facilitate my archiving of emails so much.

**Comment by rmmbr** [comment]
@Olifair: with your description I get the history back too

**Comment by Olifair** [comment]
yes, but finally I had to realise that you get the history back, not the statistical prediction pop-up list (which in my case is switched on and is the first pop-up list) ... and this is a pity.
